azure-pipelines.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       displayName: 'Use Python 3.6'
       inputs:
         versionSpec: 3.6
-    - bash: pip install wheel==0.30.0 pylint==1.9.5 flake8 requests
+    - bash: pip install wheel==0.30.0 pylint==1.9.5 flake8==3.5.0 requests
```

Comment:
Whay flake8~=3.7.0 starts to recognize feature on Python>=3.5: [Support for type hints ](https://docs.python.org/zh-cn/3/library/typing.html). Which currently used in **_src/resource-graph/azext_resourcegraph/custom.py_**. 

Enable linter detection on this will have to add another import statement `from typing import Any`.
However, `typing` is added since Python 3.5, so it won't work until we totally retire Python 2.7. Which will be soon.

Follow up is [here](https://github.com/Azure/azure-cli-extensions/issues/1141)